preprocessors/multistage-textbook-diagram.schema.json

```diff
@@ -0,0 +1,149 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "$id": "https://image.a11y.mcgill.ca/preprocessors/mutistage-textbook-diagram.schema.json",
+    "type": "object",
+    "title": "Multistage Textbook Diagram Data",
+    "description": "JSON containing pixel coordinates of different stages in a mutistage textbook diagram and the links between them",
+    "definitions": {
+        "normCoord": {
+            "description": "A pair of normalized coordinates.",
+            "type": "array",
+            "items": {
+                "type": "number",
+                "minimum": 0,
+                "maximum": 1
+            },
+            "minItems": 2,
+            "maxItems": 2
+        },
+        "contour": {
+            "type": "object",
+            "description": "A single contour of a segment as detected. Each contour is a separate region.",
+            "properties": {
+                "coordinates": {
+                    "description": "The normalized coordinates forming the contour as returned by OpenCV.",
```

Comment:
*Specifically* OpenCV?

Reply:
Copied over from semseg schema as well. I think this is intended to indicate that the origin is at the top left corner. I'll make the changes.

Reply:
That's kind of amazing since I don't think semseg uses openCV...you can just say it uses graphics coordinates.